docs/de/automatisierung-und-integration/network-monitoring/nagios.md

```diff
@@ -49,6 +49,71 @@ Die Konfigurationsdateien werden gemäß dem konfigurierten Pfad auf dem Dateisy
 
 Selbstverständlich lässt sich der Export der Dateien auch über die i-doit [Console](../cli/console/index.md) automatisieren. Eine Beschreibung der einzelnen Parameter sowie ein Beispiel sind im [entsprechenden Artikel](../cli/console/optionen-und-parameter-der-console.md#nagios-export) für die Option **Exportkonfiguration** zu finden.
 
+## CLI Console Commands und Optionen
+
+| Command     | Interne Beschreibung                                                     |
+| ----------- | ------------------------------------------------------------------------ |
+| [nagios-export](#nagios-export) | Nagios-Konfiguration exportieren |
+| [nagios-ndoutils](#nagios-ndoutils) | Ist-Zustand aus Nagios ins Logbuch schreiben |
+
+### nagios-export
+
+!!!info
```

Comment:
please use always the same syntax
!!! note "Title/text"
    or when title is set text here